content/applications/productivity/iot/config/flash_sdcard.rst

```diff
@@ -66,3 +66,28 @@ Lastly, click on :guilabel:`Flash` and wait for the process to finish.
 .. note::
    An alternative software for flashing the micro SD card is *Raspberry Pi Imager*. Download the
    *Raspberry Pi* software `here <https://www.raspberrypi.com/software/>`_.
+
+Handlers update
+===============
+
+There may be some instances where the drivers or interfacess need to be updated for individual
+devices. The IoT handlers (drivers and interfaces) code can be modified by syncing them with the
```

Comment:
"individual device" is not really clear in my opinion. You mean, by IoT box or by IoT device ? I think it can be confusing